recipes/libvips/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export LD_LIBRARY_PATH="${PREFIX}/lib:${LD_LIBRARY_PATH}"
+export CFLAGS="-O3 ${CFLAGS}"
```

Comment:
I don't think you should do `-O3`. that is a conda-forge wide choice.

Reply:
My reasoning was that libvips is marked up for the gcc autovectoriser, and that's enabled at O3 or above.  It gives a significant boost over O2 for some operations.

I'll remove it.

Reply:
is there a way to add just the one flag you want? and a comment to explain?